src/fparser/scripts/fparser2.py

```diff
@@ -102,6 +102,9 @@ def runner(_, options, args):
             print('parsing %r failed at %s' % (filename, reader.fifo_item[-1]))
             print('started at %s' % (reader.fifo_item[0]))
             return
+        except InternalError as msg:
+            print ("Internal error in fparser: {0}".format(str(msg)))
+            return
```

Comment:
The modifications to this file are not covered by the test suite. Also, rather than just `return` as if everything is fine, should we not `exit(1)` instead?